### PR TITLE
Fix Upserting temporary volume

### PR DESCRIPTION
**Pull request by hmsayem**
Signed-off-by: hmsayem <hmsayem@appscode.com>

### CODE REVIEW

**Review by piyush1146115 — APPROVED** [review]

File: pkg/cmds/server/options.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func NewExtraOptions() *ExtraOptions {
 		StashImageTag:  "",
 		MaxNumRequeues: 5,
 		NumThreads:     2,
-		ScratchDir:     "/tmp",
+		ScratchDir:     "/stash-tmp",
```
Comment (hossainemruz) [inline pkg/cmds/server/options.go:70]: Use `restic.DefaultScratchDir`

File: pkg/util/crds.go [hunk 2]
```diff
@@ -101,7 +101,7 @@ func updateStatusFunction(image docker.Docker) *api_v1beta1.Function {
 				"--endpoint=${REPOSITORY_ENDPOINT:=}",
 				"--region=${REPOSITORY_REGION:=}",
 				"--path=${REPOSITORY_PREFIX:=}",
-				"--scratch-dir=/tmp",
+				"--scratch-dir=/stash-tmp",
```
Comment (hossainemruz) [inline pkg/util/crds.go:104]: Use `fmt.Sprintf("%s",restic.DefaultScratchDir)`

File: pkg/util/crds.go [hunk 3]
```diff
@@ -135,7 +135,7 @@ func pvcBackupFunction(image docker.Docker) *api_v1beta1.Function {
...
```
Comment (hossainemruz) [inline pkg/util/crds.go:138]: Use `fmt.Sprintf("%s",restic.DefaultScratchDir)`

File: pkg/util/crds.go [hunk 4]
```diff
@@ -185,7 +185,7 @@ func pvcRestoreFunction(image docker.Docker) *api_v1beta1.Function {
...
```
Comment (hossainemruz) [inline pkg/util/crds.go:188]: Use `fmt.Sprintf("%s",restic.DefaultScratchDir)`

File: pkg/util/crds.go [hunk 5]
```diff
@@ -225,7 +225,7 @@ func pvcBackupTask() *api_v1beta1.Task {
 					Params: []api_v1beta1.Param{
 						{
 							Name:  "outputDir",
-							Value: "/tmp/output",
+							Value: "/stash-tmp/output",
```
Comment (hossainemruz) [inline pkg/util/crds.go:228]: Use `fmt.Sprintf("%s/output",restic.DefaultScratchDir)`

File: pkg/util/crds.go [hunk 6]
```diff
@@ -238,7 +238,7 @@ func pvcBackupTask() *api_v1beta1.Task {
...
```
Comment (hossainemruz) [inline pkg/util/crds.go:241]: Use `fmt.Sprintf("%s/output",restic.DefaultScratchDir)`

File: pkg/util/crds.go [hunk 7]
```diff
@@ -269,7 +269,7 @@ func pvcRestoreTask() *api_v1beta1.Task {
...
```
Comment (hossainemruz) [inline pkg/util/crds.go:272]: Use `fmt.Sprintf("%s/output",restic.DefaultScratchDir)`

File: pkg/util/crds.go [hunk 8]
```diff
@@ -282,7 +282,7 @@ func pvcRestoreTask() *api_v1beta1.Task {
...
```
Comment (hossainemruz) [inline pkg/util/crds.go:285]: Use `fmt.Sprintf("%s/output",restic.DefaultScratchDir)`

**Review by hossainemruz — CHANGES_REQUESTED** [review]

**Review by hossainemruz — APPROVED** [review]

**Review by hossainemruz — APPROVED** [review]

**Pull request merged**